rest-api/likes/models/likes-models.js

```diff
@@ -26,9 +35,21 @@ async function getLikedRecipesByUserId(id) {
     .leftJoin('recipe_images', 'recipe_images.recipe_id', 'recipes.id')
     .leftJoin('images', 'images.id', 'recipe_images.image_id')
     .where('likes.user_id', id);
+    
+    const reicpeWithLikes = [];
+
+    // .map() doesn't work with async
+    // need to find SQL query in order to optimize this
+    for(recipe of recipes){
+      reicpeWithLikes.push({
+        ...recipe,
+        ...await getLikesByRecipeID(recipe.id)
```

Comment:
Also, "likes" should just be a number! (Like the "likes" property served by GET all recipes.) 
![image](https://user-images.githubusercontent.com/8897392/73644892-250a4b80-466e-11ea-8829-4cea0e8e87cf.png)

Reply:
COUNT on potgres returns a stringified Number

Reply:
Done fixed.